src/classes/GE_GiftEntryController.cls

```diff
@@ -351,7 +346,7 @@ public with sharing class GE_GiftEntryController {
 
             return JSON.serialize(opportunityWrappers);
         } catch (Exception ex){
-            returnAuraException(ex.getMessage());
+            UTIL_AuraEnabledCommon.buildAuraException(ex.getMessage());
```

Comment:
Should this statement include the `throw` exception instead of just getting/building it?

Reply:
Probably yes ... I don't think this catch block has much utility without throwing that exception (CC @kenlewis-sfdo )

Reply:
The new line that throws looks good to me!